src/test/java/com/wikia/webdriver/testcases/discussions/PollsTests.java

```diff
@@ -16,7 +16,7 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-public class PollsTests extends NewTestTemplate {
+public class PollzsTests extends NewTestTemplate {
 
     /* TODO: Use fixtures to create Poll post */
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)